qiskit/dagcircuit/dagdepnode.py

```diff
@@ -141,6 +141,7 @@ def qargs(self, new_qargs):
         self.sort_key = str(new_qargs)
 
     @staticmethod
+    # pylint: disable=arguments-differ
```

Comment:
Were you getting a lint failure here?

Reply:
thanks for spotting this - this is a remnant of the earlier implementation attempt; removed